.github/workflows/ci_tests.yml

```
@@ -59,15 +67,35 @@ jobs:
       with:
         python-version: ${{ matrix.python }}
     - name: Install Python dependencies
-      run: python -m pip install --upgrade tox codecov
+      run: python -m pip install --upgrade tox
     - name: Run tests
-      run: tox ${{ matrix.toxargs }} -e ${{ matrix.toxenv }}
+      run: tox ${{ matrix.toxargs }} -e ${{ matrix.toxenv }} -- ${{ matrix.toxposargs }}
     - name: Upload coverage to codecov
-      if: ${{ contains(matrix.toxenv,'-cov') }}
+      if: contains(matrix.toxenv,'-cov')
       uses: codecov/codecov-action@v1
       with:
         file: ./coverage.xml
 
+  # This needs to be outside of matrix because if is not supported
+  # for only one job in a matrix. But once that is supported, this can go
+  # back to the matrix above.
+  remote_data:
+    name: Python 3.8 with all dependencies with remote data
+    if: github.event_name == 'schedule' && github.repository == 'astropy/astroquery'
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
+    - name: Install Python dependencies
+      run: python -m pip install --upgrade tox
+    - name: Run tests
+      run: tox -v -e py38-test-alldeps -- --remote-data
```

I think this would be better to be astropy dev than stable. 

```suggestion
      run: tox -v -e py38-test-alldeps-devastropy -- --remote-data
```

---

Okay, done. Thanks for the review!